Packages/tests/UTF_TestNWBExportV2.ipf

```diff
@@ -536,8 +536,23 @@ static Function TestTimeSeries(fileID, filepath, device, groupID, channel, sweep
 		WAVE/T/Z epochsSingleChannel = WaveRef(epochs, row=idx)
 		CHECK_WAVE(epochsSingleChannel, TEXT_WAVE)
 
-		WAVE/Z epochsLBN = EP_FetchEpochs(numericalValues, textualValues, sweep, params.channelNumber, params.channelType)
+		channelNumber = params.channelNumber
+		ttlBit = NaN
+
+		if(params.channelType == XOP_CHANNEL_TYPE_TTL && IsFinite(params.ttlBit))
+			WAVE/Z channelMapHWToGUI = GetActiveChannels(numericalValues, textualValues, sweep, params.channelType, TTLMode = TTL_HWTOGUI_CHANNEL)
+			CHECK_WAVE(channelMapHWToGUI, NUMERIC_WAVE)
+
+			ttlBit = log(params.ttlBit)/log(2)
+			CHECK_GE_VAR(ttlBit, 0)
+
+			channelNumber = channelMapHWToGUI[params.channelNumber][ttlBit]
```

Comment:
Please rename that here to `guiChannelNumber` as we already have `params.channelNumber` here.

Reply:
Done.